mitxpro/settings.py

```diff
@@ -808,11 +808,6 @@ def get_all_config_keys():
 CYBERSOURCE_PROFILE_ID = get_string(
     "CYBERSOURCE_PROFILE_ID", None, description="CyberSource Profile ID"
 )
-CYBERSOURCE_REFERENCE_PREFIX = get_string(
```

Comment:
It is referenced in `api_test.py` -- [Reference](https://github.com/mitodl/mitxpro/blob/master/b2b_ecommerce/api_test.py#L40). Need to update ??

Reply:
[Here](https://github.com/mitodl/mitxpro/blob/master/ecommerce/views_test.py#L89) too ???

Reply:
Thanks @asadiqbal08, I will remove those references